dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/domain/TracesFilePersistenceStrategy.kt

```diff
@@ -36,7 +40,15 @@ internal class TracesFilePersistenceStrategy(
         internalLogger
     ),
     executorService,
-    SpanSerializer(timeProvider, networkInfoProvider, userInfoProvider, envName),
+    SpanMapperSerializer(
+        LegacyToSpanEventMapper(
```

Comment:
Can we rename this class ? Legacy for me has a meaning very close to `Obsolete`, which is not the case here.